__tests__/app.test.js

```diff
@@ -4,8 +4,32 @@ const request = require("supertest");
 const db = require("../db/connection.js");
 const data = require("../db/data/test-data");
 
-describe('', ()=>{
-    test('testing', ()=>{
-        expect(1).toBe(1);
+afterAll(() => {
+    return db.end();
+  });
+  
+  beforeEach(() => {
+    return seed(data);
+  });
+
+describe('GET /api/topics', ()=>{
+    test('serves an array of all topics', ()=>{
+       return request(app).get('/api/topics').expect(200).then(({body})=>{
+        const expected = [
+            {
+              description: 'The man, the Mitch, the legend',
+              slug: 'mitch'
+            },
+            {
+              description: 'Not dogs',
+              slug: 'cats'
+            },
+            {
+              description: 'what books are made of',
+              slug: 'paper'
+            }
+          ]
+        expect(body.topics).toEqual(expected)
```

Comment:
have a look at some testing methods ie. .toMatchObject or .objectContains, as these specially test objects. 

look at expect.any() to help make your testing flexible.